modules/ismp/clients/tendermint/verifier/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "tendermint-verifier"
+version = "0.1.0"
+edition = "2021"
+description = "CometBFT light client verifier for ISMP"
+authors = ["Polytope Labs <hello@polytope.technology>"]
+publish = false
+
+[dependencies]
+# Core dependencies
+cometbft-light-client-verifier = { git = "https://github.com/cometbft/cometbft-rs", package = "cometbft-light-client-verifier" }
+cometbft = { git = "https://github.com/cometbft/cometbft-rs", features = ["rust-crypto", "secp256k1"] }
+cometbft-proto = { git = "https://github.com/cometbft/cometbft-rs", package = "cometbft-proto" }
+thiserror = "2.0.12"
+serde = { version = "1.0.219", features = ["derive"], optional = true }
+prost = "0.13"
+sha3 = "0.10"
+k256 = { version = "0.13", features = ["ecdsa"] }
```

Comment:
this library should compile for the no-std target. You'll need to turn off default features for the dependencies and add the #![no_std] tag to the library